packages/taco/src/conditions/const.ts

```diff
@@ -34,4 +34,6 @@ export const RESERVED_CONTEXT_PARAMS = [
   USER_ADDRESS_PARAM_DEFAULT,
   USER_ADDRESS_PARAM_EIP712,
   USER_ADDRESS_PARAM_EIP4361,
+  // USER_ADDRESS_PARAM_EXTERNAL_EIP4361 is not reserved and can be used as a custom context parameter
+  // USER_ADDRESS_PARAM_EXTERNAL_EIP4361
```

Comment:
Why not delete these lines instead of commenting them out?

Reply:
I thought it was more explicit (explicitly disabled), is it confusing?

Reply:
> Maybe an assertion is better.

Interesting. Where would you put the assertion?

Alternatively, maybe keep the comment on L37 and remove L38?

Reply:
An assertion could be a statement in this file that gets executed when the file is imported. That's a bit ham-fisted, I need to think about this. In the meantime, I will remove L38.

Reply:
Agreed - the assertion seems too heavy.